src/App.js

```diff
@@ -103,63 +104,179 @@ class App extends Component<Props, State> {
       return <LoginRoute login={this._dataStore.login} />;
     }
 
+    const navLinkStyle = {
+      color: '#fff',
+      filter: 'grayscale(100%)',
+      transition: 'filter 500ms ease-in-out',
+      cursor: 'pointer',
+    };
+
+    const navActiveLinkStyle = {
+      color: '#1da1f2',
+      filter: 'grayscale(0%)',
+    };
+
+    const navLinkSvgStyle = {
+      width: '2rem',
+      height: '2rem',
+      fill: 'currentColor',
+    };
+
+    const createLinkStyle = {
+      position: 'absolute',
+      bottom: '1rem',
+      right: '1rem',
+      width: '3rem',
+      height: '3rem',
+      padding: '0.75rem',
+      borderRadius: '3rem',
+      background: '#eceff1',
+    };
```

Comment:
This ^ is a good example of something that I think would be better as a shared _component_ (eg `CreateButton`) with built-in style.